custom_components/envisalink_new/alarm_control_panel.py

```diff
@@ -44,14 +47,6 @@
 
 SERVICE_CUSTOM_FUNCTION = "invoke_custom_function"
 ATTR_CUSTOM_FUNCTION = "pgm"
-ATTR_CODE = "code"
-
-SERVICE_SCHEMA = vol.Schema(
-    {
-        vol.Required(ATTR_CUSTOM_FUNCTION): cv.string,
-        vol.Optional(ATTR_CODE): cv.string,
-    }
-)
```

Comment:
`ATTR_CODE` was already defined in homeassistant.const, so I moved it, and I'm pretty sure `SERVICE_SCHEMA` was not used... These two are unrelated with my changes tho, just cleanups while I was tryting to understand the code.